handlers/comment.js

```diff
@@ -0,0 +1,366 @@
+ *         schema:
+ *          type: integer
+ *     produces:
+ *       - application/json
+ *     requestBody:
+ *       content:
+ *         application/json:
+ *          schema:
+ *           type: object   
+ *           properties: 
+ *                  text:
+ *                     type: string
+ *     security:
+ *      - bearerAuth: [] 
+ *     responses:
+ *       200:
+ *        description: comment.
+ *        content:
+ *          application/json:
+ *             schema: 
+ *                type: object
+ *                properties:
+ *                  id:
+ *                     type: integer
+ *                  user_id:
+ *                     type: integer
+ *                  tweet_id:
+ *                     type: integer
+ *                  text:
+ *                     type: string
+ *       404:
+ *          description: Impossible to comment.
+ *          content:
+ *            application/json:
+ *              schema: 
+ *                type: string
+ */
+comment.post('/tweets/:id/comments', authentication, validate([
+    param('id').isInt({ min: 1 }),
+    body('text').isLength({ max: 500 })
+]),
+    async (request, response) => {
+        try {
+            const comment = await commentController.createComment(request.token, request.params.id, request.body);
+            response.json(comment);
+        } catch (error) {
+            request.log.error(error);
+            response.end(JSON.stringify(error)).status(500);
+        }
+    });
+
+/**
+ * @openapi
+ * /comments/{id}:
+ *   put:
+ *     description: Update the comment.
+ *     tags:
+ *      - Comment
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         description: comment id
+ *         schema:
+ *          type: integer
+ *     produces:
+ *       - application/json
+ *     requestBody:
+ *       content:
+ *         application/json:
+ *          schema:
+ *           type: object   
+ *           properties: 
+ *                  text:
+ *                     type: string
+ *     security:
+ *      - bearerAuth: [] 
+ *     responses:
+ *       200:
+ *        description: Updated comment.
+ *        content:
+ *          application/json:
+ *             schema: 
+ *                type: object
+ *                properties:
+ *                  id:
+ *                     type: integer
+ *                  user_id:
+ *                     type: integer
+ *                  tweet_id:
+ *                     type: integer
+ *                  text:
+ *                     type: string
+ *       404:
+ *          description: Impossible to update the comment.
+ *          content:
+ *            application/json:
+ *              schema: 
+ *                type: string
+ */
+ comment.put('/comments/:id', authentication, validate([
+    param('id').isInt({ min: 1 }),
+    body('text').isLength({ max: 500 })
+]),
+    async (request, response) => {
+        let comment = await commentController.getComment(request.params.id);
```

Comment:
you should put this inside the try catch as well

In particular, the getComment might throw an error is the comment is not found, and it would not be catched here. Do the same in other put implementations